src/components/Item.js

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+
+// Hint: props로 id, text, onClick 등 (자유)
+export default function Item({ task, savedList, setSavedList }) {
+  const [isDone, setIsDone] = useState(false);
+  const changeState = () => {
+    setIsDone(!isDone);
+  };
+
+  const removeItem = (e) => {
```

Comment:
여기에서 props로 사용되는 `e`는 사용되지 않는 값이네요

```suggestion
  const removeItem = () => {
```